py/objarray.c

```diff
@@ -553,10 +553,24 @@ const mp_obj_type_t mp_type_bytearray = {
 #endif
 
 #if MICROPY_PY_BUILTINS_MEMORYVIEW
+#if MICROPY_CPYTHON_COMPAT
+STATIC void memoryview_attr(mp_obj_t self_in, qstr attr, mp_obj_t *dest) {
+    if (dest[0] != MP_OBJ_NULL) {
+        return;
+    }
+    if (attr == MP_QSTR_itemsize) {
+        dest[0] = mp_obj_new_int(1);
```

Comment:
This will only be correct for str/bytes/bytearray.  It should be possible to make it work also for things like `array.array('i', ...)`